### PR TITLE
French Final Update (For now)

### PR DESCRIPTION
**Pull request by ghost**
This is the final french update until new strings are added to the original file.

I fixed grammatical faults, the gallery line fault and i improved the names of certain features that sounded weird in French. All that is now fixed and cleaned.

PS: I've sent the .xml file in a XDA message to you and west. It's exactly the same changes that i've done here so no need to do it twice.

### CODE REVIEW

File: res/values-fr/resurrection_strings.xml [hunk 1]
```diff
@@ -793,7 +795,7 @@
     <string name="recent_panel_lefty_mode_title">Voir sur le bord gauche</string>
     <string name="recent_panel_lefty_mode_summary">Afficher le panneau des apps récentes sur le bord gauche</string>
     
-    <string name="slim_recent_panel_category">Récents Slim</string>
```
Comment (xenon92) [inline res/values-fr/resurrection_strings.xml:796]: This needs to stay "Récents Slim" as these options are specific for Slim Recents. No problem, we'll fix it after we merge this pull request.

File: res/values-fr/resurrection_strings.xml [hunk 2]
```diff
@@ -66,8 +66,10 @@
     <string name="github_omni">https://github.com/omnirom/</string>
     <string name="github_xenon92">https://github.com/xenon92</string>
     <string name="rr">ROM Resurrection Remix ®</string>
-    <string name="rr_website_title">Forum Resurrection Remix</string>
+    <string name="rr_website_title">Site Web Resurrection Remix</string>
```
Comment (xenon92) [inline res/values-fr/resurrection_strings.xml:69]: TODO: Make this "Website"

(we'll fix it Madikan)

**Comment by xenon92** [comment]
Thank you for the fixes. We'll merge it soon.

**Comment by westcripp** [comment]
308 line error . must be like d\'applications

**Comment by westcripp** [comment]
i got ton of errors on latest translations

**Comment by ghost** [comment]
Sorry for the d\'applications error, thanks for fixing it (my bad :( ) 
But it seems that all the rest of the translation is clean and it should work. Can you tell me what errors showed up ?

**Comment by westcripp** [comment]
yup no problem .i just compiling rom we wll see .

**Comment by ghost** [comment]
I just saw something that "could" create errors. Line 308, there is a space between "" and the first string.

**Comment by westcripp** [comment]
ok now all working

**Pull request merged**